docs/api.md

```diff
@@ -0,0 +1,266 @@
+# API
+
+## Consumer
+
+```haskell
+Consumer :: Object -> { start, stop }
+```
+
+Initializes a consumer with an object of options, returning an interface with `start` and `stop` functions.  A consumer processes ordered messages from a single [category](), keeping track of its position to resume processing after restarts.  Once finished reading a category, it continues to poll for new messages.
+
+?> Consumers put the "autonomy" in event-sourced autonomous services, because they operate as independent actors.  If one goes down, the rest can presumably stay up and running (unless you [host them together](/api?id=starting-stopping)).
+
+Consumers can either act either as [components](), which process messages and write new events, or [aggregators](), which process events to update [read models]().  From an implementation standpoint, the main difference that is aggregators will often use an `init` function to initialize the read model, while components will not.
+
+### Options / Example
+
+| Option | Type | Default | Description |
+| - | - | - | - |
+| `batchSize` | `Number` | `1000` | Number of messages to retrieve per batch
+| `category` | `String` | | Name of the category to read |
+| `handlers` | `Object` | `{}` | Map of message types to async handler functions |
+| `groupMember` | `Number` | | Optional group identifier to enable concurrency |
+| `groupSize` | `Number` | | Optional group total to enable concurrency |
+| `init` | `Function` | | Optional read model initializer |
+| `name` | `String` | | Unique name used to store the consumer's position |
+| `positionUpdateInterval` | `Number` | `100` | Minimum number of messages processed before the current position is recorded |
+| `tickInterval` | `Number` | `100` | Milliseconds to wait after latest batch before polling for new messages |
```

Comment:
I'm not sure I understand.  This already renders as a table?

Reply:
Makes it easier for use wierdos to read in the CLI.

Reply:
You mean pad the cells with whitespace?  I tend not to do that in markdown for the same reason we tend not to do it in long lists of `require` statements.  But if it would help you, I can.